### DISCUSSION TITLE
Newbie confusion about where to download the right ASIO driver.

### DISCUSSION THREAD

**Discussion by gene96817** [jamulussoftware/jamulus#1229] [General]
I was helping a chorus get a few of their windows users running. Although they started with Jamulus.io, they were confused with the web page for downloading the ASIO driver. Two of the users ended up paying $29 and was ensnared in a lot of trashware promotion. The most savy Win user found the right link at the bottom of the webpage.

Since I am not a Win user, I don't see very many of these websites pushing trashware or worst. How do we help new Jamulus users from falling victim to these schemes?

**Comment by ann0see** [comment]
The new website will link the binary directly (we're hosting it). Maybe you could contact the developer and tell him about these ads on his website.

**Reply by ann0see** [reply]
Edit: just saw that a new version has just popped up. I will try it very soon. I HOPE that this has improved a few bugs and a few of the requests I made are included. But looking at the changelog makes me feel as if they weren't unfortunately ;-(.

Wait a moment: in fact after looking at the manual he DID include some of my requests. This sounds great!

**Reply by gene96817** [reply]
I can't wait for the new website to go live.

**Reply by ann0see** [reply]
I saw you opened and deleted a post about A4A 2.15 beta. For me it seems to work better. It even has an online manual now
